### PR TITLE
#45 Improve custom shortcodes

### PR DESCRIPTION
**Pull request by vitojeng**
required by #45

### CODE REVIEW

File: content/en/docs/master/custom_stream.md [hunk 1]
```diff
@@ -272,12 +272,12 @@ Below we provide some examples that demonstrate how to develop your own
 stream applications. More description of each example could be found in
 javadoc.
 
-- [WordCount]({{< url-ohara-code >}}/ohara-stream/src/test/java/oharastream/ohara/stream/examples/WordCountExample.java): 
+- [WordCount]({{< ohara-file "ohara-stream/src/test/java/oharastream/ohara/stream/examples/WordCountExample.java" >}}):
```
Comment (vitojeng) [inline content/en/docs/master/custom_stream.md:275]: {{ ohara-file $file }} 會自動根據 url 裡的 branch name, 轉換成 ohara repo 對應的 branch file 的 url.

Comment (chia7712) [reply content/en/docs/master/custom_stream.md:275]: 能給個範例嗎

Comment (vitojeng) [reply content/en/docs/master/custom_stream.md:275]: 以 line 275 為例, 這篇文章 custom_streams.md 放在 content/en/docs/master/ 內, 屬於 `master` branch.
故產生結果為: 
* https://github.com/oharastream/ohara/blob/master/ohara-stream/src/test/java/oharastream/ohara/stream/examples/WordCountExample.java

但若這篇文章完全不修改, 改放在 content/en/docs/0.10.x/ 的話, 就屬於 `0.10.x` branch. 產生結果為:
* https://github.com/oharastream/ohara/blob/0.10.x/ohara-stream/src/test/java/oharastream/ohara/stream/examples/WordCountExample.java

Comment (chia7712) [reply content/en/docs/master/custom_stream.md:275]: 了解，感謝解釋！

**Pull request merged**